### PR TITLE
Mecha light tweaks

### PR DESCRIPTION
**Pull request by brndd**
I always thought the mecha lights were too dim (they're dimmer than a lantern), so I made them brighter. Now they're brighter than a lantern, and even brighter still on the Firefighter Ripley. The Deathsquad Ripley's brightness has been reduced from 60 radius to 12, because the old value made it look like the lighting system was broken.

I also made active mechs (that is, ones with a pilot in them) emit a low amount of light even when the main lights are off. It's supposed to represent cabin lights leaking out, or something. The combat mechs do not have this to allow for stealth operations.

~~Oh yeah, and I removed the special cursor combat mechs have, because I've always found it supremely annoying and it's kinda glitchy with some interface elements. I hope there isn't some deeper purpose to it that eludes me.~~ ~~I put the cursor on a toggle option, defaulting to off. As a side effect of this, it's no longer exclusive to combat mechs (because working in separate interface shit for them would've been a pain).~~ I removed that from this PR and will make a new one for it.

Comparisons: [old Ripley lights]
image
(https://i.imgur.com/oNXHtfN.png) 

[new Ripley lights]
image
(https://i.imgur.com/4AAX78g.png)

[Ripley ambient lights]
image
(https://i.imgur.com/UdIN2wx.png)

[a lantern]
image
(https://i.imgur.com/LLkcSlP.png)

:cl:
* tweak: Mech lights are now brighter and have more range.
* tweak: Deathsquad Ripley's lights now only illuminate 150% of the screen radius (was 750%)
* tweak: Non-combat mechs now produce a small amount of light when active even when the lights are off.

### CODE REVIEW

**Comment by SonixApache** [comment]
RETURN THE CURSOR
OR SUFFER MY CURSE

**Comment by brndd** [comment]
I'll see if I can't do a toggle option for the cursor.

**Comment by brndd** [comment]
I put the cursor on a toggle option, defaulting to off for non-combat mechs and on for combat mechs. As a side effect of this, it's no longer exclusive to combat mechs (because working in separate interface shit for them would've been a pain).

**Comment by brndd** [comment]
...and the cursor thing has been reverted from this PR. I'll be making a new one for it.

**Comment by Reviire** [comment]
Why on earth would you nerf the deathripley

**Comment by brndd** [comment]
>Why on earth would you nerf the deathripley

Because the default light radius on it was so absurdly high that it was well past the point of diminishing returns. The reduced value of 12 is still higher than any other mech, and is enough to provide bright illumination at a range four tiles beyond the edge of the screen. The old value would go about 52 tiles beyond the edge of the screen, which is just kinda confusing and very unlikely to ever be actually useful.

Here's what the modified light radius on the deathsquad Ripley looks like. You can just barely see the light start fading in the top left corner of the screen:

dripley

File: code/game/mecha/mecha.dm [hunk 1]
```diff
@@ -1109,9 +1112,11 @@
 		return
 	lights = !lights
 	if(lights)
-		set_light(luminosity + lights_power)
+		light_power = light_brightness_on
```
Comment (jknpj) [inline code/game/mecha/mecha.dm:1115]: Try using this instead

```
light_power = lights ? light_brightness_on : light_brightness_off
set_light(light_power)
```

Comment (brndd) [reply code/game/mecha/mecha.dm:1115]: Power and range are two different variables. Power affects the "intensity" of the light, ie. how brightly it illuminates tiles within its range, but does not extend the range. Range is the number of tiles the light extends outwards from its source. For a visual comparison, [this](https://user-images.githubusercontent.com/17928298/43481207-b47991fe-94db-11e8-9b0a-258ecc079e29.png) is range 6 power 1, and [this](https://user-images.githubusercontent.com/17928298/43481294-ef54d96e-94db-11e8-9c48-46ff976adb94.png) is range 6 power 2 (and a different hue).

Comment (jknpj) [reply code/game/mecha/mecha.dm:1115]: I'm just dumb

**Pull request merged**